### ISSUE TITLE
False positive with class attribute string value and docstring

### ISSUE THREAD

**Issue by btmills** [10sr/flake8-no-implicit-concat#86]
Thank you for building this plugin! It's saved me from several bugs :tada:

I found what I believe to be a false positive when a class attribute has a string value and a docstring:

**Version**

I cloned the repository as of 7db5b51bb5a28586a47c39ca4184d459b29e620a, followed the `pipenv` instructions, and added a new `tests/run_flake8/attribute_docstring.py` file.

**Input**

```py
class Class:
    attribute = "value"
    """docstring"""
```

**Actual**

```
./attribute_docstring.py:2:24: NIC002 Implicitly concatenated str literals over multiple lines
```

**Expected**

I'd expect no issues to be reported because `"value"` and `"""docstring"""` are not concatenated:

```
Python 3.12.6 (main, Sep  6 2024, 19:03:47) [Clang 15.0.0 (clang-1500.3.9.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> class Class:
...     attribute = "value"
...     """docstring"""
...
>>> Class.attribute
'value'
```

Unfortunately I'm insufficiently familiar with the Python language standard to point to language that specifies the behavior I'm observing, and my Googling found [documentation](https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation) that doesn't mention attribute docstrings as an exception.

**Comment by 10sr** [comment]
Thanks for reporting the issue!

**Comment by btmills** [comment]
I've upgraded the dependency and was able to remove the temporary `# noqa: NIC002`. Thanks for fixing!
